src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.8 BLOCK -->
+
+## Security
```

Comment:
The security file was added by a bot in [this protobuf PR](https://github.com/Azure/azure-functions-language-worker-protobuf/pull/91) and seems like the changes haven't been pulled over since.

Reply:
Discussed with @brettsam and this shouldn't be necessary